### ISSUE TITLE
URLs shown incorrectly

### ISSUE THREAD

**Issue by The-Compiler** [akrennmair/newsbeuter#264]
I'm on the current git head (03d3b2d) and noticed since some while some URLs are broken. I'm on Archlinux and using newsbeuter with TinyTinyRSS.

For example, with [this feed](http://www.cyanogenmod.org/feed), when showing the article "Introducing the Cyanogen Apps Package" I see:

```
Feed: news - Cyanogenmod
Title: Introducing the Cyanogen Apps Package
Author: bsxtn
more-themes-added-regularly
Date: Wed, 11 Nov 2015 01:20:37 +0100
```

The other articles look fine.

With a [Google Alert](https://www.google.com/alerts) feed, I see something like this for every post:

```
Feed: qutebrowser - Google Alert
Title: The-Compiler/qutebrowser
sg=AFQjCNHMzf_FR-_GPL8L3qKPru-gKBpIkg
Date: Tue, 02 Feb 2016 11:45:44 +0100
```

**Comment by Minoru** [comment]
Hm… I couldn't reproduce any of those.

Can you `make clean`, then rebuild and re-try? Maybe you somehow manager to compile with an older object file.

Does Ctrl-L help?

**Comment by The-Compiler** [comment]
I've seen this multiple times in the past, and now suddenly I can't reproduce it either... I'll follow up with more information when I find out something - looks like it doesn't happen when opening a fresh instance.

**Comment by The-Compiler** [comment]
I could reproduce this again, immediately after a fresh start, and multiple times.

Then I played with "open in browser" and "save file" a bit, and now it shows up correctly again, ever after a restart.

Ctrl-L didn't help when it happened, and "open in browser" opened the correct URL.

**Comment by The-Compiler** [comment]
I noticed when I see `jCNHiJPoFKa4ncUuosGcHo7HsumBokg` in newsbeuter, and do `|less`, the link shows up like this:

```
Link: https://www.google.com/url?rct=j&sa=t&url=http://osdir.com/ml/general/2016-02/msg30353.html&ct=ga&cd=CAIyHGY3ZTkzZTVmOTNhZWY3Y2M6Y29tOmVuOlVTOkw&usg=AFQ
jCNHiJPoFKa4ncUuosGcHo7HsumBokg 
```

When I resize my terminal to fullscreen, suddenly I see the correct link. So it seems to me like only the last line is painted if a link line-wraps?

**Comment by Minoru** [comment]
Bisecting shows I introduced this regression in 26e47fc.

**Comment by The-Compiler** [comment]
I can confirm this is fixed - thanks @Minoru! :tada:

**Comment by Minoru** [comment]
Thank @Duncanla for prodding me with his PR :) Now let's see if I can keep rolling and fix #282 as well!
